client/README.md

```diff
@@ -10,9 +10,11 @@ Two current options are to [access Matrix and contact a bot](https://wiki.polkad
 
 ## Development
 
-To develop you need two env variables:
+To develop you need three env variables:
 
-- `PUBLIC_CAPTCHA_KEY`: The [reCaptcha v2 site key](https://www.google.com/u/0/recaptcha/admin).
+- `PUBLIC_CAPTCHA_PROVIDER`: The captcha provider. Currently `procaptcha` and `recaptcha` are supported. You will then need one of the following site keys:
+    - `PUBLIC_PROSOPO_SITE_KEY`: The [Prosopo site key](https://prosopo.io/) which is `5HUBceb4Du6dvMA9BiwN5VzUrzUsX9Zp7z7nSR2cC1TCv5jg`.
```

Comment:
Yes, this is generated by us but you can generate your own by registering on our website. I think it would be best if we hop on a call to go through everything next week. I'll reach out to you on Element to arrange.